### PR TITLE
Fix bugs part 2

### PR DESCRIPTION
**Pull request by Electr1Xx**

### CODE REVIEW

File: src/components/UsernameDialog/UsernameDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import { avatarUrl, coverUrl } from 'src/constants/images.constants';
+
+export const UsernameDialog = () => {
+	const { profile, usernameVisible, changeVisible } = useContext(AuthContext);
+	const usernameFormRef = useRef<null | HTMLFormElement>(null);
+	const [isLoading, setIsLodaing] = useState(false);
+	const { editPersonalInfo } = useGilder();
+
+	const handleSubmit = async (data: { username: string }) => {
+		setIsLodaing(true);
```
Comment (Slmii) [inline src/components/UsernameDialog/UsernameDialog.tsx:18]: type in setter

File: src/views/Forum/components/TrendingPostsCard.tsx [hunk 2]
```diff
@@ -186,7 +190,7 @@ export const TrendingPostsCard = () => {
 											</Box>
 											<P2Title1> {differenceDate(Number(post.created_at))} </P2Title1>
 										</Box>
-										<Box sx={{ cursor: 'pointer' }} onClick={() => navigate(`/forum/postindividual/${post.idx}`)}>
+										<Link component={NavLink} to={`/forum/postindividual/${post.idx}`} sx={{ textDecoration: 'none' }}>
```
Comment (Slmii) [inline src/views/Forum/components/TrendingPostsCard.tsx:193]: perhaps set the textDecoration as default to `none` in theming

**Review by Slmii — CHANGES_REQUESTED** [review]

File: src/components/UsernameDialog/UsernameDialog.tsx (hunk 1 above)
Comment (Electr1Xx) [reply src/components/UsernameDialog/UsernameDialog.tsx:18]: added

File: src/views/Forum/components/TrendingPostsCard.tsx (hunk 2 above)
Comment (Electr1Xx) [reply src/views/Forum/components/TrendingPostsCard.tsx]: fixed

**Review by Slmii — APPROVED** [review]

**Pull request merged**